src/spec.c

```diff
@@ -1934,7 +1935,7 @@ void ReindexPool_ThreadPoolDestroy() {
 #ifdef FTINFO_FOR_INFO_MODULES
 void IndexSpec_AddToInfo(RedisModuleInfoCtx *ctx, IndexSpec *sp) {
   char *temp = "info";
-  char name[strlen(sp->name) + strlen(temp) + 2];
+  char name[sp->nameLen + strlen(temp) + 2];
```

Comment:
strlen(temp) is known beforehand. I guess we can remove that second strlen call @ashtul. agree?

Reply:
From what I know, the compiler optimizes these static strings and converts the command into a literal at compile time.
I usually prefer to leave it this way though I think maybe to remove the `temp` pointer and have the `"info"` string explicitly.
WDYT?